### PR TITLE
fix inconsistent protobuf client ids

### PR DESCRIPTION
**Pull request by TarikViehmann**
A combination of an uninitialized `long int` client id and its conversion to an `unsigned int` before asserting the corresponding protobuf clips fact can potentially lead to broken client ids in clips. Avoid this by sticking to `long int` and initializing the client id increment value.

### CODE REVIEW

File: src/libs/protobuf_clips/communicator.h [hunk 1]
```diff
@@ -217,7 +217,7 @@ class ClipsProtobufCommunicator
 	  sig_peer_sent_;
 
 	fawkes::Mutex map_mutex_;
-	long int      next_client_id_;
+	long int      next_client_id_ = 0;
```
Comment (morxa) [inline src/libs/protobuf_clips/communicator.h:220]: For consistency's sake, this should be initialized in the constructor(s), not in the header. All the other members are also initialized in the constructors.

**Review by morxa — CHANGES_REQUESTED** [review]
Good catch!

**Review by morxa — APPROVED** [review]
Thanks for the fix!

**Comment by TarikViehmann** [comment]
Thanks for the review :+1:

**Pull request merged**